Creados los modelos de Operacion, UnidadTiempo y TipoOperacion y la r…

juliodion12345 opened the pull request:
…elación entre ellos.

Aqui esta el codigo en el que he trabajado, por favor revisarlo para saber que elementos le faltan @lfmexi :D

juliodion12345 committed:
Creados los modelos de Operacion, UnidadTiempo y TipoOperacion y la relación entre ellos

lfmexi commented on common/models/operacion.js:
```diff
@@ -0,0 +1,10 @@
+    }
+    next();
+  });
+
```
@juliodion12345 No deberíamos tener también una fecha de modificación de la operación?

juliodion12345 replied:
@lfmexi Las operaciones pueden modificarse tambien, voy a agregar ese campo y agregar la creacion de esa fecha

lfmexi commented on common/models/tipo-operacion.json:
```diff
@@ -0,0 +1,18 @@
+{
```
@juliodion12345 Viste si estas están expuestas como RESTFUL services?

juliodion12345 replied:
Si estan expuestas como RESTFUL services @lfmexi

lfmexi committed:
Merge branch 'develop' of github.com:lmyjo/InvestmentsUtility into feature-ui3

Conflicts:
	server/model-config.json

lfmexi committed:
Cambios en el modelo de operaciones.

Se ha cambiado el modelo de operaciones, además, se ha agregado
la relación entre projecto y operación, además de las restricciones,
por lo que el IU6 y el IU7 quedan cubiertos parcialmente.

lfmexi commented on server/bin/autoupdate.js:
```diff
@@ -0,0 +1,38 @@
+var app = require('../server');
```
@juliodion12345 vamos a tener que cambiar este archivo, ya que no tenemos más los modelos de unidades de tiempo  y de tipos de operación.

juliodion12345 replied:
@lfmexi Ya los cambie y agregue la información de los modelos que faltaban (Owner y project)

lfmexi commented on server/bin/automigrate.js:
```diff
@@ -0,0 +1,92 @@
+var app = require('../server');
```
@juliodion12345 vamos a tener que cambiar este archivo, ya que no tenemos más los modelos de unidades de tiempo y de tipos de operación.

juliodion12345 replied:
@lfmexi Ya los cambie y agregue la información de los modelos que faltaban (Owner y project)

juliodion12345 committed:
Resolved merge conflicts

lfmexi commented:
@juliodion12345 para el siguiente feature será mejor que hagamos tests para cubrirnos las espaldas. Haré el merge a develop.
